recipe/meta.yaml

```diff
@@ -11,30 +11,29 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<35]
+  skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
   run:
+    - backports.zoneinfo  # [py<39]
     - cached-property
     - defusedxml >=0.6.0
-    - dnspython >=1.14.0
+    - dnspython >=2.0.0
     - isodate
     - lxml >3.0
     - oauthlib
     - pygments
     - python
-    - python-dateutil
-    - pytz
     - requests >=2.7
     - requests_ntlm >=0.2.0
-    - requests-kerberos
     - requests-negotiate-sspi  # [win]
     - requests-oauthlib
+    - tzdata
```

Comment:
Unless this uses the time zone database files directly, you'd probably want `python-tzdata` (or, if that's meant as a dependency for `backports.zoneinfo`, see https://github.com/conda-forge/staged-recipes/pull/13014#issuecomment-717758465).

Reply:
The package declares a dependency on the [`tzdata` pip package](https://pypi.org/project/tzdata/)
https://github.com/ecederstrand/exchangelib/blob/master/setup.py#L55

Is the `conda` equivalent `python-tzdata`?

Reply:
> *Is the conda equivalent `python-tzdata`?*

Looks like it - thanks @mbargull!